pegasus/sites.v3/hourofcode.com/styles/060-hoc-specific.css

```diff
@@ -103,4 +178,14 @@ label.has-error {
   .front-header-banner h4 {
       font-size: 16px;
   }
-}
\ No newline at end of file
+  #hoc-header-branding {
+    width: 6em;
+    position: relative;
+    top: -1em;
+    left: 0;
+    float: left;
+  }
+  #language-dropdown {
+    margin-left: 1em;
+  }
+}
```

Comment:
Sure! Will probably need a bit more guidance on when it makes sense to pull out styling into its own file. I'll find you to chat more about this offline when I come in!